intercom_flutter/android/build.gradle

```diff
@@ -39,6 +39,15 @@ android {
     if (project.android.hasProperty('namespace')) {
         namespace 'io.maido.intercom'
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_17
+    }
```

Comment:
Update to

```
compileOptions {
    sourceCompatibility JavaVersion.VERSION_1_8
    targetCompatibility JavaVersion.VERSION_1_8
}

kotlinOptions {
    jvmTarget = '1.8'
}
```

Ref: https://github.com/firebase/flutterfire/blob/eb9c7874dc7e34afba1c2fd0c02c6d56bd6adfc0/packages/firebase_core/firebase_core/android/build.gradle#L36